shell/platform/windows/platform_handler.cc

```diff
@@ -205,6 +206,20 @@ int ScopedClipboard::SetString(const std::wstring string) {
 
 }  // namespace
 
+// Indicates whether an exit request may be canceled by the framework.
+// These values must be kept in sync with ExitType in platform_handler.h
+static const char* kExitTypeNames[] = {"required", "cancelable"};
```

Comment:
Maybe move this closer to (or inside of) `RequestAppExit`, since that's the only place it is used.

You could also define the contents of the array in terms of `kExitTypeCancelable` and `kExitTypeRequired`, since they're constexprs. You might also be able to make this a constexpr. It would be one less thing to keep in sync.

Reply:
I can use the constants in the array, but it does not appear that this can be set to a `constexpr`.